test/specRunner.js

```diff
@@ -0,0 +1,10 @@
+console.log(`RUNNING ALL TESTS`);
+console.log(`------------------------------------------------------`);
+console.log(``);
+
```

Comment:
EBI - consider naming conventions at the start, as I decided to redistribute my tests into new files half way through for clarity as my original naming convention became confusing as more tests were added.